block/manager.go

```diff
@@ -100,10 +109,11 @@ func NewManager(
 		executor:    exec,
 		dalc:        dalc,
 		retriever:   dalc.(da.BlockRetriever), // TODO(tzdybal): do it in more gentle way (after MVP)
-		HeaderOutCh: make(chan *types.Header),
-		HeaderInCh:  make(chan *types.Header),
-		blockInCh:   make(chan *types.Block),
-		retrieveCh:  make(chan uint64),
+		daHeight:    s.DAHeight,
+		HeaderOutCh: make(chan *types.Header, 10),
+		HeaderInCh:  make(chan *types.Header, 100),
+		blockInCh:   make(chan *types.Block, 100),
+		retrieveCh:  make(chan interface{}, 1000),
 		syncCache:   make(map[uint64]*types.Block),
```

Comment:
Please add comment why these channels are buffered and why the numbers make sense.